kubectl-toolbox/kubectl-toolbox

```diff
@@ -48,7 +50,7 @@ usage() {
     echo "      -v run in verbose mode"
     echo "      -i [docker image to run as toolbox]"
     echo "      -c [entrypoint command to run in toolbox container, defaults to 'bash']"
-    echo "      -n [name for privileged pod, defaults to 'toolbox']"
```

Comment:
Can you add an extra argument for the pod name as well so this isn't lost? Maybe `-p`